src/components/datafile/item.tsx

```diff
@@ -15,7 +16,18 @@ export const DataFileListItem: FC<DataFileListItemProps> = memo(function Wrapped
 }) {
 
 	const onTriggerDelete = useCallback(() => {
-		onDelete(dataFile);
+		modals.openConfirmModal({
+			title: "Delete Sample Dependency",
+			centered: true,
+			children: (
+				<Text size="sm">
+					Are you sure you want to delete the sample dependency { `"${dataFile.fileName}"` } from the runner?
+				</Text>
+			),
+			labels: { confirm: "Delete", cancel: "Cancel" },
+			confirmProps: { color: "red" },
+			onConfirm: () => onDelete(dataFile)
+		});
```

Comment:
I think we should rename `Sample Dependency` to something else.. `Audio File` ?
Both here and in the left menu to get to the sample dep page.
They're not actually "Dependencies".. they are simply files that can be applied to a buffer (aka dataref) mapping.